docs/content/guides/getting-started/grid-size.md

```diff
@@ -16,17 +16,15 @@ searchCategory: Guides
 
 # Grid size
 
-[[toc]]
-
-## Overview
+Set the width and height of the grid, columns, and rows, using either absolute values or values relative to the parent container.
```

Comment:
This page is only about setting the dimensions of the grid (not columns or rows), so I am not sure why columns or rows are mentioned on this page.

Reply:
@warpech – good point. I've modified it here: [407d219](https://github.com/handsontable/handsontable/pull/10046/commits/407d219c0eb25dbcff8a81c77420c3bb2229277e)